mode-apis-base/src/alertingModels.ts

```diff
@@ -226,53 +226,36 @@ export enum AlertState {
     INEFFECTIVE = 4,
 }
 
+export interface Alert {
+    readonly id: string;
 
-export interface AlertEvaluation {
-    // Snapshot of an AlertCondition taken when it is evaluated.
-    readonly condition: AlertCondition;
-    readonly description: string;
-    readonly entityId: string;
-    readonly severity: AlertSeverity;
-}
+    readonly projectId: number;
 
+    // Alert Status can transition in the following manner:
+    // ALERTING -> RECOVERED or INEFFECTIVE or OBSOLETE
+    readonly state: AlertState;
 
-export interface MetricsAlertEvaluation extends AlertEvaluation {
+    readonly invocationTimestamp: string;
+    readonly recoveryTimestamp?: string;
+}
+
+export interface MetricAlert extends Alert {
```

Comment:
Using `Metric` instead of `Metrics` because the alert is associated with a specific "metric".